client/my-sites/upgrades/domain-management/transfer/index.jsx

```diff
@@ -17,6 +17,15 @@ class Transfer extends React.Component {
 	render() {
 		const slug = this.props.selectedSite.slug,
 			domainName = this.props.selectedDomainName;
+
+		let transferToAnotherUser = null;
```

Comment:
@rralian I'm heading in that direction and using `_.get` to guard against undefined options